engine/src/module-engine.cpp

```diff
@@ -674,8 +674,25 @@ static hash_t __MCScriptObjectHash(MCValueRef p_value)
 
 static bool __MCScriptObjectDescribe(MCValueRef p_value, MCStringRef& r_description)
 {
-    r_description = MCSTR("<script object>");
-    return true;
+    __MCScriptObjectImpl *self;
+    self = (__MCScriptObjectImpl *)MCValueGetExtraBytesPtr(p_value);
```

Comment:
Per the style guide, please use RAII and C++-style casts, i.e.

```c++
auto self = reinterpret_cast<__MCScriptObjectImpl*>(MCValueGetExtraBytesPtr(p_value));
```

We allow `auto` here because the type is explicit.